### ISSUE TITLE
Implement automated accessibility testing in the CI of all SciTools repos

### ISSUE THREAD

**Issue by trexfeathers** [SciTools/.github#214]
We should explore Linux-based tools for automatically testing accessibility of our GitHub content. These should be runnable manually and also integrated into our CI. Testable things would include: not skipping Markdown heading levels, presence of alt-text for images, safe use of emojis.
